src/cmd-encrypt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+# Used to encrypt files before upload (such as build.log)
+
+import subprocess
+import argparse
+import os
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("cert", help="X.509 Certificate")
+parser.add_argument("input", help="Path to file to encrypt into PKCS7 structure")
+parser.add_argument("output", help="Path to encrypted PKCS7 structure")
+args = parser.parse_args()
+
+files = [args.cert, args.input]
+for path in files:
+    if not os.path.exists(path):
+        raise FileNotFoundError(path)
+
+subprocess.check_call(["openssl", "smime", "-encrypt", "-aes-256-cbc", "-in", args.input,
```

Comment:
The container for COSA is pretty large. I'm not opposed to adding `openssl` but would be happier using Gnutls's `certtool`.

Reply:
In @saqibali-2k's defense, that suggestion came from me. :)  I thought maybe we already had `openssl` in there so I thought it'd be worth trying but I hadn't actually checked. We definitely already have `openssl-libs`.

OK, looking at a recent CI build, we *don't* currently pull in `openssl`, but we *do* pull in `gnutls-utils`, so I guess that's a vote in favour. OTOH, I see locally `openssl` is about 1M which is super tiny, so at those numbers, I'd value ergonomics and whatever @saqibali-2k is more comfortable with.

Reply:
I was having some trouble using `certtool` in the pipeline.  I believe it was also returning `command not found`, I can double check.

Reply:
Upon further review...openssl is the easier tool. We could do some stuff with `pkcstool`. But using `openssl` is easier.